src/score_hv/harvesters/daily_bfg.py

```diff
@@ -2,32 +2,35 @@
 
 import os
 import sys
-from pathlib import Path
-from collections import namedtuple
-from dataclasses import dataclass
-from dataclasses import field
-from datetime import datetime as dt
-
+import ast
 import xarray as xr
 import numpy as np
 import cftime
+from pathlib import Path
 from netCDF4 import MFDataset
+from datetime import datetime as dt
+from collections import namedtuple
+from dataclasses import dataclass
+from dataclasses import field
+from score_hv.config_base  import ConfigInterface
+from score_hv.stats_utils  import var_stats
+from score_hv.region_utils import GeoRegions
 
-from score_hv.config_base import ConfigInterface
-from score_hv import stats_utils
 
 HARVESTER_NAME = 'daily_bfg'
 VALID_STATISTICS = ('mean', 'variance', 'minimum', 'maximum')
 
+
 """Variables of interest that come from the background forecast data.
 Commented out variables can be uncommented to generate gridcell weighted
 statistics but are in development and are currently not fully supported.
 """
-VALID_VARIABLES  = (#'icetk', # sea ice thickness (m)
+VALID_VARIABLES  = (
+                    #'icetk', # sea ice thickness (m)
                     #'lhtfl_ave', # surface latent heat flux (W m^-2)
-                    'netrf_avetoa', # top of atmosphere net radiation flux (SW and LW) (W m^-2)
-                    #'prate_ave', # surface precip rate (mm weq. s^-1)
-                    'prateb_ave', # bucket surface precip rate (mm weq. s^-1)
+                    'netrf_avetoa',#top of atmosphere net radiative flux (SW and LW) (W/m**2)
+                    'netR',#surface energy balance (W/m**2)
```

Comment:
trying to adhere to convention:
```suggestion
                    'netef_ave', # surface energy budget (W/m**2)
```